### DISCUSSION TITLE
Problem with the rounding of the bar labels

### DISCUSSION THREAD

**Discussion by EarlvanEick** [nmalkin/plot-likert#43] [Q&A]
Thank you very much for this library! 

Unfortunately I have a problem with the rounding of the bar labels. I have the following aggregated dataframe: 
	        1.0	          2.0	           3.0
Q12.1_Tri	59.53	26.51	13.95
Q12.2_Tri	61.99	24.89	13.12
Q12.3_Tri	68.84	25.58	5.58
Q12.4_Tri	45.37	18.98	35.65
Q12.5_Tri	61.40	26.98	11.63

The columns are already percentages, but the problem with the rounding of the bar labels happens with individual data and absolute figures as well.

This is my code: plot_likert.plot_counts(data2,  another_scale, compute_percentages=False, bar_labels=True);

![Download](https://github.com/nmalkin/plot-likert/assets/68033866/cd75ae49-6409-45fb-993c-224feaac290f)

As you can see, the bar labels are truncated and not rounded. The same happens if I set compute_percentages as True.

Is this a bug or is did I make a mistake?

**Comment by nmalkin** [comment]
(Moved to #44)
